guidelines/README.md

```diff
@@ -32,76 +32,74 @@ Some other multimedia projects compilations:
 
 ## Purpose
 
-Arwes is an intent to materialize futuristic science fiction concepts in a toolset
-to let you create interfaces that inspire advanced technology, especially spatial,
+Arwes intends to materialize futuristic science fiction concepts in a toolset
+you may use to create interfaces inspired by advanced technology, especially spatial,
 influenced by alien conceptions.
 
-Connected, augmented, and energized are the feelings sought in the experiences
-provided with projects built with Arwes. So the interfaces are intensive and rich.
+Connected, augmented, and energized are the feelings sought in the experiences provided by projects built with Arwes, so Arwes interfaces are intensive and rich.
```

Comment:
Can you set the character line limit to 80 please?

Reply:
👍